### PR TITLE
Add level gameplay mode

### PR DESCRIPTION
**Pull request by harrisonMondragon**

### CODE REVIEW

**Comment by harrisonMondragon** [comment]
Pls note that I needed to take the endlevel overlay and port all of it into levels +page.svelte. This was because I wanted the levels page to control when the overlay was open. I also used gameplay.ts as a mock database for the time being

**Comment by Colin-C32** [comment]
> Pls note that I needed to take the endlevel overlay and port all of it into levels +page.svelte. This was because I wanted the levels page to control when the overlay was open. I also used gameplay.ts as a mock database for the time being

Does Svelte not allow us to pass the isOpen argument?

**Comment by harrisonMondragon** [comment]
I'm sure there is a way to do that but I couldn't figure it out in a reasonable amount of time. I also needed to control the overlay button on this page which made it even more weird to figure out. See playNextLevelClicked(), I'm not sure how to do this logic in the end screen overlay

**Comment by harrisonMondragon** [comment]
> I'm sure there is a way to do that but I couldn't figure it out in a reasonable amount of time. I also needed to control the overlay button on this page which made it even more weird to figure out. See playNextLevelClicked(), I'm not sure how to do this logic in a separate component

File: wikiclue/src/routes/levels/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        </div>
+    </div>
+    <div class="search-container">
+        <p class = "sub-header">Find a Wikipedia page that contains the following words:</p>
+        <div class="words-container">
+            <p class="search-words">{wordsToFind[0]}</p>
+            <p class="search-words">{wordsToFind[1]}</p>
+        </div>
+        {#if !$levelOver}
+            <input type="text" class="search-bar" placeholder="Enter the Wikipedia URL here..." bind:value={searchTerm}/>
```
Comment (Kjeon7) [inline wikiclue/src/routes/levels/+page.svelte:103]: Let's change this since we're not doing urls anymore

**Review by Kjeon7 — CHANGES_REQUESTED** [review]
Just one comment to be addressed other than that looks LGTM

File: wikiclue/src/routes/levels/+page.svelte (hunk 1 above)
Comment (harrisonMondragon) [reply wikiclue/src/routes/levels/+page.svelte]: done

**Review by Kjeon7 — APPROVED** [review]

**Pull request merged**